devimint/src/external.rs

```diff
@@ -678,16 +670,7 @@ pub async fn open_channels_between_gateways(
             .open_channel(gw_b, sats_per_side * 2, Some(sats_per_side))
             .await?;
 
-        if let Some(txid) = txid {
-            bitcoind.poll_get_transaction(txid).await?;
-        }
```

Comment:
I'm a bit confused how the destructuring suddenly isn't necessary anymore, I didn't find changes to `open_channel`'s or `poll_get_transaction`'s type signature when searching the diff. What am I missing?

Reply:
Thanks for the question! Helped me find some documentation I made stale.

I changed the return type of `open_channel` from `Result<Option<Txid>>` to `Result<Txid>`. The gateway wouldn't return a txid until `v0.5`, but we now guarantee returning a txid of the funding tx.